webpack.common.js

```diff
@@ -6,24 +6,34 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 const process = require('process');
 const dotenv = require('dotenv');
+const getServiceURL = require('./scripts/webpack/getServiceURL');
 
 const envFileVars = dotenv.config().parsed;
 
 const makeEndpoints = () => {
   const defaults = {
     HAPPA_API_ENDPOINT: 'http://localhost:8000',
     HAPPA_PASSAGE_ENDPOINT: 'http://localhost:5001',
+    PROXY: undefined,
```

Comment:
Is this a leftover?
```suggestion
    HAPPA_PROXY_INSTALLATION: undefined,
    HAPPA_PROXY_BASE_DOMAIN: undefined,
```

Reply:
Yeah, doing a quick little refactoring pass